docs/reference-manual/native-image/JFR.md

```diff
@@ -11,23 +11,20 @@ redirect_from: /reference-manual/native-image/JFR/
 JDK Flight Recorder (JFR) is an event recorder for capturing information about a JVM, and an application running on the JVM. 
 GraalVM Native Image supports building a native executable with JFR events, and users can use [`jdk.jfr.Event` API](https://docs.oracle.com/en/java/javase/20/docs/api/jdk.jfr/jdk/jfr/Event.html) with a similar experience to using JFR in the Java HotSpot VM.
 
-To record JFR events when running a native executable, JFR support and JFR recording must be enabled. 
 
-## Add JFR Support and Record Events at Run Time
-
-To build a native executable with the JFR events support, you first need to include JFR at build time, then enable the system, start a recording, and configure logging at native executable run time.
+## Include JFR Support at Build Time and Record Events at Run Time
+To record JFR events when running a native executable, JFR support must be added to the image at build time and a recording must be started at runtime.
 
 To build a native executable with JFR, use the `--enable-monitoring=jfr` flag:
 ```shell
 native-image --enable-monitoring=jfr JavaApplication
 ```
-To enable the system, start a recording, and configure logging at run time, the following options are supported:
+To start a recording, and configure logging at run time, the following options are supported:
 
-* `-XX:+FlightRecorder`: use to enable JFR
```

Comment:
Should it be also removed in the user guide guides/build-and-run-native-executable-with-jfr.md?
See https://github.com/oracle/graal/pull/7595/files#diff-2da5e6c4769171a5b5d09723d7e149f19c68dfa0bb6998699f6b04498066e088R20

Reply:
Yes I think it should. I'll update those as well.